### PR TITLE
Update rerun_auth_config to the not deprecated form

### PR DESCRIPTION
**Pull request by rmohr**
Get rid of

```
{"component":"unset","file":"prow/config/config.go:1234","func":"k8s.io/test-infra/prow/config.(*Config).validateComponentConfig","level":"warning","msg":"rerun_auth_config will be deprecated in July 2020, and it will be replaced with rerun_auth_configs['*'].","severity":"warning","time":"2020-08-26T08:35:32Z"}
```

Signed-off-by: Roman Mohr <rmohr@redhat.com>

### CODE REVIEW

**Review by dhiller — APPROVED** [review]
/lgtm
/approve

**Comment by dhiller** [comment]
Funny, check-prow-config fails. I remember having tried that also. Maybe our config check is too old?

**Comment by rmohr** [comment]
/hold

ok, let's hold this one for the moment.

File: github/ci/prow/files/config.yaml [hunk 1]
```diff
@@ -48,8 +48,9 @@ deck:
         - podinfo.json
   tide_update_period: 1s
   rerun_auth_config:
```
Comment (dhiller) [inline github/ci/prow/files/config.yaml:50]: it should be `rerun_auth_configs` here: https://github.com/kubernetes/test-infra/pull/16957/commits/42d4861750244594aff8e14d57be8392bf03b119

**Review by dhiller — CHANGES_REQUESTED** [review]

File: github/ci/prow/files/config.yaml (hunk 1 above)
Comment (rmohr) [reply github/ci/prow/files/config.yaml]: Ah, nice. Fixed.

**Review by dhiller — APPROVED** [review]
/lgtm

**Pull request merged**